apps/server/src/modules/learnroom/controller/cards.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Controller, Get, NotImplementedException, Query } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { ICurrentUser } from '@shared/domain';
+import { Authenticate, CurrentUser } from '@src/modules/authentication/decorator/auth.decorator';
+import { CardIdsParams } from './dto';
+
+export class CardListResponse {}
+
+@ApiTags('Cards')
+@Authenticate('jwt')
+@Controller('cards')
+export class CardsController {
+	@Get()
+	getCards(@CurrentUser() currentUser: ICurrentUser, @Query() cardIdParams: CardIdsParams): Promise<CardListResponse> {
```

Comment:
should an endpoint like this exist? We want cards in board / column, in that case boardId / columnId would make more sense, instead of having a list of cards already from front-end